crypto/async/arch/async_win.c

```diff
@@ -34,7 +34,7 @@ void async_local_cleanup(void)
 
 int async_fibre_init_dispatcher(async_fibre *fibre)
 {
-    fibre->fibre = ConvertThreadToFiber(NULL);
+    fibre->fibre = ConvertThreadToFiberEx(NULL, FIBER_FLAG_FLOAT_SWITCH);
```

Comment:
Strictly speaking we still support Windows XP (in that we've never, as far as I can remember, formally decided to stop supporting it). My guess is we don't compile on it any more (but possibly not). But this will definitely break it. Probably I'm ok with that, but I wonder what @openssl/committers think.

Reply:
I can wrap the call inside `defined(_WIN32_WINNT) && _WIN32_WINNT >= 0x600` if that's a problem. We already do this in `providers/implementations/rands/seeding/rand_win.c`.

Reply:
Let's do that